src/TestFramework/PhpUnit/CommandLine/ArgumentsAndOptionsBuilder.php

```diff
@@ -21,7 +21,6 @@ public function build(string $configPath, string $extraOptions): array
         return array_filter([
             '--configuration',
             $configPath,
-            '--stop-on-failure',
```

Comment:
Why do we need to continue after a failure?

Reply:
we don't, this flag is added on XML level (covered by tests btw), and this is just a duplication that I found